### ISSUE TITLE
Can't be installed since recent Moonraker path changes

### ISSUE THREAD

**Issue by Guilouz** [mainsail-crew/moonraker-timelapse#71]
Timelapse can't be installed directly on a fresh install of Klipper and Moonraker.

/klipper_config folder doesn't exist anymore and have been replaced by /printer_data/config
```
pi@speeder-pad:~$ bash ~/moonraker-timelapse/install.sh
Klipper service found! Stopping during Install.
Moonraker service found! Stopping during Install.
Linking extension to moonraker...
ERROR: /home/pi/klipper_config not found.
Try:
Usage: /home/pi/moonraker-timelapse/install.sh -c /path/to/klipper_config
Exiting...
```

Need to be installed with:
```
/home/pi/moonraker-timelapse/install.sh -c /home/pi/printer_data/config
```

**Comment by FrYakaTKoP** [comment]
known issue, will be fixed soon. Thanks for reporting anyways

**Comment by alxpettit** [comment]
I imagine `ln -s ~/printer_data/config/ ~/klipper_config` would work as a workaround.

**Comment by AndyJones2003** [comment]
Hi,
I'm installing on a laptop using Kiauh.

I'm using the latest Mainsail 2.4.1. and adding two instances to support two printers . I've managed to resolve the issue with the 'New' path structure for multipe printers and have Timelapse installed. The last action is to add [timelapse] to the moonraker.conf (This should give me an entry in LHS menu for Timelapse)  When I do this I get 
[moonraker.py:add_warning()] - Unparsed config section [timelapse] detected.  
I installed Moonraker as a single printer install using Kiauh on the same laptop and successfully ran Timelapse on my system and recorded a video.


[klippy.log](https://github.com/mainsail-crew/moonraker-timelapse/files/10223410/klippy.log)
[moonraker.log](https://github.com/mainsail-crew/moonraker-timelapse/files/10223411/moonraker.log)

**Comment by PigeonFX** [comment]
Hello, I see the script still has the old path /klipper_config, is the current best practice to run the script and then do a symbolic link as per @alxpettit recommendation of running:

`ln -s ~/printer_data/config/ ~/klipper_config`

Or will this course some sort of conflict when the script is updated to the newer /printer_data/config path?

**Comment by AndyJones2003** [comment]
I did get this working on a multi instance of moon raker eventually and now producing timelapse of my prints. Sorry if my terminology is wrong, first time working with Linux,  but I worked out the symbolic links all seemed fine but when checking the links I noticed the timelapse.py seemed to be what I would call a ‘shortcut’ in windows (in WinSCP it has a red arrow-like shortcut symbol?) I removed these ‘shortcuts’ and replaced with the actual timelapse.py and It all came to life!

Rightly or wrongly this is how I did it.

**This is all based on the back of [FrYakaTKoP](https://github.com/FrYakaTKoP)  work and threads, Thanks [FrYakaTKoP]**(https://github.com/FrYakaTKoP)  all credit to you...

Only setup timelapse for my first printer instance as that is all I needed.

This was on a laptop running Linux mint with two instances (for two printers) setup with Kiauh.

Crowsnest for the USB camera Logitech C270

Run following to Initialise USB ports, (Worked on mine Google it if not)

sudo systemctl mask brltty-udev.service 
sudo systemctl stop brltty-udev.service

My symbolic’s are;

ln -sf "/home/pi/moonraker-timelapse/component/timelapse.py" "/home/andy/moonraker-timelapse/component/timelapse.py"
ln -sf "/home/pi/moonraker-timelapse/klipper_macro/timelapse.cfg" "/home/andy/Ender3V2_data/config/timelapse.cfg"
ln -sf "/home/pi/klipper_config/timelapse.cfg" "/home/andy/Ender3V2_data/config/timelapse.cfg"

These will need adapting to the naming of your data folders. 

andy/Ender3V2_data would be [your Linux session name]/[your moonaker instance name]_data

in WinSCP if you login in the home folder you will see the name of the instance and data folders

So..

Do a git get for timelapse as per the install.

Don’t bother running linstall.sh it will fail trying to find the instance of Klipper.

Apply the symbolics. The install.sh stops Klipper and Moonraker when applying these but I didn’t do this?

Validate the files in the symbolic statements paths exist and are ‘true’ instances of the files not ‘shortcuts’.

Add the includes ect in Printer.cfg and Moonraker.conf as per the original installation/configuration.md

Update Moonraker.conf with 
[timelapse]
output_path: ~/Ender3V2_data/timelapse
frame_path: ~/Ender3V2_data/timelapse/tmp

(comment out/delete any other entries these are managed in the web page timelapse settings.)


Reboot the system..
